lib/rubocop/cop/rspec/multiple_expectations.rb

```diff
@@ -31,6 +31,26 @@ module RSpec
       #     end
       #   end
       #
+      # @example with aggregate_failures (defaults to true)
+      #
+      #  # not flagged by rubocop
```

Comment:
"No offense" or just "good", or "good - the cop ignores when RSpec aggregates failures".

Reply:
Updated with change.